src/schema/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
         Generic: require('./generic'),
         FlightBooking: require('./flight-booking'),
         VacationRental: require('./vacation-rental'),
+        AutomobileInsurance: require('./automobile-insurance'),
```

Comment:
I know Automobile sounds more sophisticated, but I think that rules out motorbikes whereas we'd probably include them under this banner. I'd rename to `MotorInsurance`

Reply:
I insure my motorcycle and the flow is different. The post-deeplink might be the same.